ffbs-parker-nodeconfig/files/lib/gluon/upgrade/906-parker-nodeconfig

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/lua
+
+local site = require 'gluon.site'
+local uci = require('simple-uci').cursor()
+
+site_config_server = site.parker.config_server()
+uci:section('parker', 'nodeconfig', 'nodeconfig',
+    { config_server = site_config_server }
+)
+uci:save('parker')
+
+site_config_pubkey = site.parker.config_pubkey()
```

Comment:
it would be nice, if we can also add the pubkey to uci so that it's easier to override it on-demand.

```suggestion
local site_config_server = site.parker.config_server()
local site_config_pubkey = site.parker.config_pubkey()
uci:section('parker', 'nodeconfig', 'nodeconfig',
	{
		config_server = site_config_server,
		config_pubkey = site_config_pubkey
	}
)
uci:save('parker')
```

Reply:
@SmithChart still open

Reply:
See https://github.com/SmithChart/community-packages/pull/5

Reply:
@grische can you verify that this has been resolved by https://github.com/SmithChart/community-packages/pull/5 ?